presto-cli/src/main/java/io/prestosql/cli/TableNameCompleter.java

```diff
@@ -58,10 +58,18 @@ public TableNameCompleter(QueryRunner queryRunner)
 
     private List<String> listTables(String schemaName)
     {
-        return queryMetadata(format("SELECT table_name FROM information_schema.tables WHERE table_schema = '%s'", schemaName));
+        String[] array = schemaName.split("\\.");
```

Comment:
How does it work for schemas with a dot in their name?
(for example, in tpch, tpcds connectors, schemas can have dot in their name (although it doesn't fully work now: https://github.com/prestosql/presto/issues/1005))

Reply:
Oh, I didn't care about that case. Fixed to allow schema name includes dot.